### PR TITLE
Feature/post comments

### PR DESCRIPTION
**Pull request by Vlad-Gheorghiu**

### CODE REVIEW

File: src/app/features/posts/components/post-comments/post-comments.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.card-comments {
+    margin: 16px;
+}
```
Comment (lolismatic) [inline src/app/features/posts/components/post-comments/post-comments.component.scss]: Missing enter at the eof.

File: src/app/features/posts/components/post-details/post-details.component.html [hunk 2]
```diff
@@ -3,4 +3,17 @@
...
+        <input matInput placeholder=" Leave a comment " name="text" color="primary" ngModel>
+      </mat-form-field>
+      <button mat-raised-button type="submit" color="accent">Add</button>
+    </form>
+
+    <hyn-post-comments
+      class="post-comments"
+      [comments]="post.comments">
+    </hyn-post-comments>
+</div>
```
Comment (lolismatic) [inline src/app/features/posts/components/post-details/post-details.component.html]: Missing enter at eof

File: src/app/features/posts/components/post-comments/post-comments.component.scss (hunk 1 above)
Comment (Vlad-Gheorghiu) [reply src/app/features/posts/components/post-comments/post-comments.component.scss]: Done

File: src/app/features/posts/components/post-details/post-details.component.html (hunk 2 above)
Comment (Vlad-Gheorghiu) [reply src/app/features/posts/components/post-details/post-details.component.html]: Done

**Pull request merged**